File: patterns/c-patterns/pattern62.c
// Pattern 62: ...

// # * # * #
//  * # * #
//   # * #
//    * #
//     #

#include <stdio.h>

int main()
{
    int i, j;
    int n = 5;

    for (i = n; i >= 1; i--)
    {
        for (j = n; j >= 1; j--)
        {
            if (i >= j)
            {
                if (j % 2 == 0)
                    printf("* ");
                else
                    printf("# ");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern62.py
# Pattern 62: ...

# # * # * #
#  * # * #
#   # * #
#    * #
#     #

n = 5

for i in range(n, 0, -1):
    for j in range(n, 0, -1):
        if i >= j:
            if j % 2 == 0:
                print("* ", end="")
            else:
                print("# ", end="")
        else:
            print(" ", end="")
    print()
